packs/metaweather/README.md

```diff
@@ -0,0 +1,8 @@
+# MetaWeather Integration Pack
+
+Pack providing integration with MetaWeather API.
+
+## Actions
+
+* ``get_location_results`` - Retrieve a list of locations given a search query. Ex. ZIP code, Country/City/State, etc.
```

Comment:
If possible, since both actions return a dictionary, it would be great if you can include an example of the result object in the readme.